.github/workflows/test-spras.yml

```diff
@@ -57,6 +57,10 @@ jobs:
       with:
         # Choose version from https://github.com/apptainer/apptainer/releases
         apptainer-version: 1.3.6
+    - name: Pull local-neighorhood Docker image
+      run: docker pull gli527/local-neighborhood:latest
+    - name: Build local-neighborhood Docker image
+      run: docker build -f docker-wrappers/LocalNeighborhood/Dockerfile -t gli527/local-neighborhood:latest .
```

Comment:
This should be placed (and respectively modified) in `build-containers.yml` instead